components/admin_console/database/migrations_table.tsx

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import classNames from 'classnames';
+
+import {FormattedMessage} from 'react-intl';
+
+import {SchemaMigration} from 'mattermost-redux/types/admin';
+
+import './migrations_table.scss';
+import {Client4} from 'mattermost-redux/client';
+
+export type Props = {
+    createHelpText: React.ReactElement;
+    className?: string;
+}
+
+type State = {
+    migrations: SchemaMigration[];
+}
+
+class MigrationsTable extends React.PureComponent<Props, State> {
+    constructor(props: Props) {
+        super(props);
+        this.state = {
+            migrations: [],
+        };
+    }
+
+    componentDidMount() {
+        Client4.getAppliedSchemaMigrations().then((result) => {
```

Comment:
The Client4 should be accessed by dispatching a redux action instead of using it directly. I don't think this data should be stored in redux state since this is the only component that will care about it, but we should still do it for consistency